data/Yushi5058.json

```
@@ -1,17 +1,19 @@
 {
 	"name": "Yushi",
 	"type": "personal",
-	"bio": "Self-taught web developer, passionate about the tech industry and constantly improving my skills to advance my career.",
-	"links": [{
-			"name": "Discord",
-			"url": "discord.com/users/1071758711330193478",
-			"icon": "FaDiscord"
-		},
+	"bio": "Self taught web dev | Front-end web dev | Linux & Bash Enthusiast",
+	"tags": ["Open Source", "Front-End", "Linux", "Javascript"],
+	"socials": [
+		{"icon": "FaDiscord", "url":"discord.com/users/1071758711330193478"},
+		{"icon": "FaGithub", "url":"https://github.com/Yushi5058"},
+		{"icon": "SiCodewars", "url": "https://www.codewars.com/users/Yushi5058"}
+		
+	],
+	"links": [
 		{
-			"name": "Github",
+			"name": "Follow me on Github",
 			"url": "https://github.com/Yushi5058",
 			"icon": "FaGithub"
-		}
-	]
+	],
```

There is a missing bracket `}` & extra comma `,` which is leading the `json` check fail.

```suggestion
	}
     ]
```

---

Thank you for pointing that for me. I'll resolve it :)